MhLabs.AwsSignedHttpClient/ServiceCollectionExtension.cs

```diff
@@ -13,7 +13,8 @@ public static class ServiceCollectionExtension
     {
         private static readonly Random _jitterer = new Random();
 
-        public static IServiceCollection AddSignedHttpClient<TClient, TImplementation>(this IServiceCollection services, string baseUrl = null, bool useCircuitBreaker = true) where TClient : class
+        public static IServiceCollection AddSignedHttpClient<TClient, TImplementation>(this IServiceCollection services, string baseUrl = null, bool useCircuitBreaker = true,
```

Comment:
I wonder if we should pass an options object instead? Just for future discussion - no need to change now, but it can get messy with too many optional parameters

Reply:
I agree and considered that but decided no since it would be a breaking change, will discuss it separately